modules/overviews/app/controllers/overviews/overviews_controller.rb

```diff
@@ -49,6 +76,50 @@ def update_project_custom_values
       respond_to_with_turbo_streams(status: service_call.success? ? :ok : :unprocessable_entity)
     end
 
+    def project_life_cycles_sidebar
+      render :project_life_cycles_sidebar, layout: false
+    end
+
+    def project_life_cycles_dialog
+      respond_with_dialog(
+        ProjectLifeCycles::Sections::EditDialogComponent.new(@project)
+      )
+    end
+
+    def project_life_cycles_form
+      service_call = ::ProjectLifeCycleSteps::PreviewAttributesService
+                .new(user: current_user,
+                     model: @project,
+                     contract_class: ProjectLifeCycleSteps::UpdateContract)
+                .call(permitted_params.project_life_cycles)
+
+      update_via_turbo_stream(
+        component: ProjectLifeCycles::Sections::EditComponent.new(service_call.result),
+        method: "morph"
+      )
+      # TODO: :unprocessable_entity is not nice, change the dialog logic to accept :ok
+      # without dismissing the dialog, alternatively use turbo frames instead of streams.
```

Comment:
While true, I think it is safe to say that this is outside of the scope of this PR 👍🏻